File: cloud-common/src/test/java/com/sequenceiq/cloudbreak/TestException.java
package com.sequenceiq.cloudbreak;

public class TestException extends RuntimeException {
    public TestException(Throwable cause) {
        super(cause);
    }

    public TestException(String message) {
        super(message);
    }

    public TestException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: core/src/test/java/com/sequenceiq/cloudbreak/TestException.kt
package com.sequenceiq.cloudbreak

class TestException : RuntimeException {
    constructor(cause: Throwable) : super(cause) {
    }

    constructor(message: String) : super(message) {
    }

    constructor(message: String, cause: Throwable) : super(message, cause) {
    }
}
